src/mango/src/mango_cursor.erl

```diff
@@ -46,7 +46,7 @@
 
 create(Db, Selector0, Opts) ->
     Selector = mango_selector:normalize(Selector0),
-    UsableIndexes = mango_idx:get_usable_indexes(Db, Selector, Opts),
+    UsableIndexes = mango_idx:get_usable_indexes(Db, Selector0, Opts),
```

Comment:
This looks suspicious but it was these way before the reverted commit see https://github.com/apache/couchdb/commit/d0445f53a837db374e56eb59ede63694ed6e4c33